### PR TITLE
admin guide systemd improvements

### PR DESCRIPTION
**Pull request by jfaltenbacher**
### Description

Please check my proposals.

### To which product versions do the changes apply?

The first column is to be filled by the requester, the second column is to be filled by the doc team after the changes have been backported to the maintenance branches.

| Code 15     | Backport Done
| ----------  | ----------
| [ x ] 15 SP3  | *(no backport necessary)*
| [ x ] 15 SP2  | [ ]
| [ ] 15 SP1  | [ ]
| [ ] 15 SP0  | [ ]

| Code 12     | Backport Done
| ----------  | ----------
| [ ] 12 SP5  | [ ]
| [ ] 12 SP4  | [ ]
| [ ] 12 SP3  | [ ]
| [ ] 12 SP2  | [ ]

### CODE REVIEW

File: xml/systemd.xml [hunk 1]
```diff
@@ -14,61 +14,58 @@
   </dm:docmanager>
  </info>
  <para>
-  The program &systemd; is the process with process ID 1. It is responsible for
-  initializing the system in the required way. &systemd; is started directly by
+  The program &systemd; is responsible for initializing the system in the
+  required way and has the process ID 1. &systemd; is started directly by
```
Comment (dmpop) [inline xml/systemd.xml:18]:
```suggestion
  &systemd; is responsible for initializing the system, and it and has the process ID 1. &systemd; is started directly by
```

File: xml/systemd.xml [hunk 2]
```diff
@@ -14,61 +14,58 @@
...
-  aggressively paralleling service starts.  Furthermore, systemd only starts a
+  The main advantage of &systemd; is that it considerably speeds up boot time by
+  aggressively paralleling service starts. Furthermore, &systemd; only starts a
   service when it is really needed. Daemons are not started unconditionally at
-  boot time, but rather when being required for the first time. systemd also
-  supports Kernel Control Groups (cgroups), snapshotting and restoring the
-  system state and more. See <link
-  xlink:href="http://www.freedesktop.org/wiki/Software/systemd/"/> for details.
+  boot time, but when being required for the first time. &systemd; also
+  supports Kernel Control Groups (cgroups), snap-shooting and restoring the
```
Comment (dmpop) [inline xml/systemd.xml:29]: snapshotting is correct. Although _creating snapshots_ is better.

File: xml/systemd.xml [hunk 3]
```diff
@@ -14,61 +14,58 @@
...
+<para>
+    &systemd; is a system and session manager for Linux, compatible with System V
+    and LSB init scripts. 
+    The main features of &systemd; are:
+</para>
+    <itemizedlist mark="bullet" spacing="normal">
     <listitem>
      <para>
-      provides aggressive parallelization capabilities
+      aggressive parallelization capabilities
```
Comment (dmpop) [inline xml/systemd.xml:48]: _Aggressive_ sounds like a subjective qualifier. Better remove it.

File: xml/systemd.xml [hunk 4]
```diff
@@ -14,61 +14,58 @@
...
-   <title>What is systemd</title>
-   <para>
-    systemd is a system and session manager for Linux, compatible with System V
-    and LSB init scripts. The main features are:
-   </para>
-   <itemizedlist mark="bullet" spacing="normal">
+<para>
+    &systemd; is a system and session manager for Linux, compatible with System V
+    and LSB init scripts. 
+    The main features of &systemd; are:
```
Comment (dmpop) [inline xml/systemd.xml:43]:
```suggestion
    The main features of &systemd; include the following.
```

File: xml/systemd.xml [hunk 5]
```diff
@@ -14,61 +14,58 @@
...
     <listitem>
      <para>
-      keeps track of processes using Linux cgroups
+      tracking of processes using Linux cgroups
      </para>
     </listitem>
     <listitem>
      <para>
-      supports snapshotting and restoring of the system state
+      snap-shooting and restoring of the system state
```
Comment (dmpop) [inline xml/systemd.xml:68]: _snapshotting_ is correct. _creating snapshots_ is better.

File: xml/systemd.xml [hunk 6]
```diff
@@ -83,19 +80,21 @@
...
    </para>
-   <itemizedlist mark="bullet" spacing="normal">
+   <para>
+    <quote>Unit file</quote> is a generic term used by &systemd; for the 
+    following:
+   </para>
+    <itemizedlist mark="bullet" spacing="normal">
     <listitem>
      <formalpara>
       <title>Service</title>
```
Comment (dmpop) [inline xml/systemd.xml:100]: There is a itemized list with a single item? Perhaps replacing this with a single sentence would be better? For example: _ In &systemd; terminology, unit files refer to services_

File: xml/systemd.xml [hunk 7]
```diff
@@ -215,14 +215,14 @@
...
+    If the output goes to a terminal (and not to a pipe or a file, for example),
+    &systemd; commands send long output to a pager by default. Use the
     <option>--no-pager</option> option to turn off paging mode.
    </para>
    <para>
-    systemd also supports bash-completion, allowing you to enter the first
+    &systemd; also supports bash-completion, allowing you to enter the first
     letters of a subcommand and then press <keycap function="tab"/> to
-    automatically complete it. This feature is only available in the
+    autocompletion. This feature is only available in the
```
Comment (dmpop) [inline xml/systemd.xml:225]: ~~to autocompletion~~

File: xml/systemd.xml [hunk 8]
```diff
@@ -1224,9 +1224,9 @@ Startup finished in 2666ms (kernel) + 21961ms (userspace) = 24628ms</screen>
    <sect3 xml:id="sec-boot-systemd-debug-complete">
     <title>Review the complete start-up process</title>
     <para>
-     The above-mentioned commands let you review the services that started and
-     the time it took to start them. If you need to know more details, you can
-     tell &systemd; to verbosely log the complete start-up procedure by
+     The above-mentioned commands let you review the services that are started
```
Comment (dmpop) [inline xml/systemd.xml:1227]:
```suggestion
     The commands above let you review the services that are started 
```

**Review by dmpop — COMMENTED** [review]
A few comments here and there.

File: xml/systemd.xml (hunk 2 above)
Comment (jfaltenbacher) [reply xml/systemd.xml:29]: oh yes, it take creating snapshots then

File: xml/systemd.xml (hunk 3 above)
Comment (jfaltenbacher) [reply xml/systemd.xml:48]: that was also my thought. Will also remove it from the upper section then

File: xml/systemd.xml (hunk 6 above)
Comment (jfaltenbacher) [reply xml/systemd.xml:99]: there are more list items following

File: xml/systemd.xml (hunk 7 above)
Comment (jfaltenbacher) [reply xml/systemd.xml:224]: removed

**Pull request merged**